controllers/client/round_tripper.go

```diff
@@ -33,11 +34,17 @@ func NewInstrumentedRoundTripper(relatedResource string, metric *prometheus.Coun
 		relatedResource: relatedResource,
 		wrapped:         transport,
 		metric:          metric,
+		headers:         map[string]string{"user-agent": "grafana-operator/" + embeds.Version},
```

Comment:
Let's move to a variable, so the code would follow the same style as with other properties and it'll be slightly easier to read.